### ISSUE TITLE
Release Statuses - Add unknown as an option

### ISSUE THREAD

**Issue by ItsME6969** [Lidarr/Lidarr#3731]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

If a release is marked as unknown in MusicBrainz, the release will not appear on Lidarr as Lidarr only supports 4 other release status. Ofcourse one solution is to edit the metadata on MusicBrainz but that is not realistic when adding hundreds of artists and not even knowing which releases are missing to begin with because lidarr shows `no indication` of the missing releases unless you specifically know which releases are missing and try to find them on MB. I know this is gonna be closed and limited as the other threads supporting the idea have but I just had to put it out there because I dont think forcing people to edit the metadata is really the solution here.

### Describe the solution you'd like

Add `unknown` as an option that is unchecked by default

### Describe alternatives you've considered

Maybe even add it as an `Advanced` Feature

### Anything else?

Bottom line, Add unknown as an option. Thanks

**Comment by mscreations** [comment]
I am going to add this as a request as well. Too many album releases are listed as unknown and Lidarr does not show them available even though Picard finds it without an issue.

**Comment by h9rst** [comment]
Are there any news on this or maybe a Script resolution to bulk edit the release Status on musicbrainz?

**Comment by Millio345** [comment]
+1 for this request. I understand not wanting to have tons of unwanted/incorrect results in Lidarr, but especially when directly manually searching by MB id ("lidar:RELEASE-GROUP-ID"), it should be possible to add those.

**Comment by Sketch6307** [comment]
Seriously, I have so many albums that there's just not a match for at all, and several more that there's no match fitting the tracks I have. And editing the metadata is *not* the correct answer because I have no knowledge of what the correct status of a release is. I'm honestly not sure why it was chosen to support only a subset of the possible release statuses, but they should all be supported.

**Comment by lazee486** [comment]
so usenet/torrent downloads them and then they all sit waiting for me to go manually import each album one by one...and all im doing is editing release group and typing unknown...because i have no idea which release group did it, nor do i care... I have the track artist album, format and everything else....but i cant import it for a field that has no use in my method...

does anyone know a faster way im missing to handle this?

subnote i run mergerfs, if a drive dies the media disappears and lidarr re requests it...and then i have to suddenly do all the imports again :(

**Comment by bakerboy448** [comment]
@lazee486 Lidarr Release Group is not and has never been required for imports.  What you describe has nothing to do with the MusicBrainz release status.


For support and inquiries, please use our [Discord](http://lidarr.audio/discord) channel. GitHub is designated solely for bug reports and feature requests. It seems that this issue may fall under a support request, so we kindly ask you to visit our Discord for assistance. Thank you.

**Comment by IxeYgrek** [comment]
> Are there any news on this or maybe a Script resolution to bulk edit the release Status on musicbrainz?

Hi,
GPT Codex made this to bypass the problem: https://github.com/IxeYgrek/Lidarr.AllReleases

Seem not working with official Lidarr cache server. Need local LidarrAPI.metadata cache server (https://hub.docker.com/r/blampe/lidarr.metadata).

**Comment by augustuen** [comment]
> > Are there any news on this or maybe a Script resolution to bulk edit the release Status on musicbrainz?
> 
> Hi,
> GPT Codex made this to bypass the problem: https://github.com/IxeYgrek/Lidarr.AllReleases

Doesn't work with the official metadata/cache server.

**Comment by IxeYgrek** [comment]
> > > Are there any news on this or maybe a Script resolution to bulk edit the release Status on musicbrainz?
> > 
> > 
> > Hi,
> > GPT Codex made this to bypass the problem: https://github.com/IxeYgrek/Lidarr.AllReleases
> 
> Doesn't work with the official metadata/cache server.

Thanks for the feedback, I'll correct in the readme.
But honestly I think it should work if the official cache was correctly synchronized.
